src/Uno.Extensions.Reactive.UI/Presentation/Bindings/Bindable.cs

```diff
@@ -12,18 +13,20 @@ namespace Uno.Extensions.Reactive.Bindings;
 /// An helper class use to data-bind a value.
 /// </summary>
 /// <typeparam name="T">The type of the value</typeparam>
-public class Bindable<T> : IBindable, INotifyPropertyChanged
+public class Bindable<T> : IBindable, INotifyPropertyChanged, IFeed<T>
 {
 	/// <inheritdoc />
 	public event PropertyChangedEventHandler? PropertyChanged;
 
-	private T? _value;
+	private T _value = default!; // This is going to be init by property.Subscribe(OnOwnerUpdated);, and anyway with bindings we cannot ensure non-null!
```

Comment:
If it can't be non-null, why keep it non-nullable?

Reply:
Otherwise I'm not able to re-expose it as `IFeed<T>` (would be `IFeed<T?>`)